stimuli-ci-generation/cloudbuild.yaml

```diff
@@ -0,0 +1,59 @@
+steps:
+- id: 'build'
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - build
+  - --rm
+  - --no-cache
+  - --tag=gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID
+  - .
+
+# Pushing the intermediate image can be useful to debug test failures.
+- id: 'intermediate-push'
+  waitFor: ['build']
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - push
+  - gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID
+
+# Verify 'convert' (ImageMagick) is available.
+- id: 'test-convert'
+  waitFor: ['intermediate-push'] # Otherwise the test failure should stop push.
+  name: 'gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID'
+  args: ['convert', '--version']
+
+- id: 'test'
+  waitFor: ['intermediate-push'] # Otherwise the test failure should stop push.
+  name: 'gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    set -e
+    Rscript test_build.R
+    # Verify expected test_build.R output is present.
+    [ -s plot1.png ]
+    echo "ok"
+
+- id: 'retag'
+  waitFor: ['test', 'test-convert']
+  name: 'gcr.io/cloud-builders/docker'
+  args:
+  - tag
+  - gcr.io/$PROJECT_ID/rcran-build:$BUILD_ID
+  - gcr.io/kaggle-images/rcran:$_TAG
```

Comment:
I see kaggle mentioned in a few places. What do we use kaggle for?

Reply:
Nothing.